src/desktop/apps/artwork/components/commercial/query.coffee

```diff
@@ -10,6 +10,7 @@ module.exports = """
     contact_message
     is_price_range
     price
+    shippingInfo
```

Comment:
This does require Metaphysics to be deployed to production since otherwise GraphQL will complain that this field doesn't exist.

Reply:
Yea, all the above is true:)

MP is on staging so that will work on staging but MP deploy should be before this is deployed. So maybe wait till that to merge this PR. If you are cool with the change I can merge after MP deployment.